samigo/samigo-app/src/webapp/jsf/author/editAssessment.jsp

```diff
@@ -349,6 +353,8 @@ $(window).load( function() {
         	<f:param value="#{partBean.poolNameToBeDrawn}"/>
         	<f:param value="#{partBean.randomQuestionsDrawDate}"/>
         	<f:param value="#{partBean.randomQuestionsDrawTime}"/>
+        	<f:param value="#{partBean.poolOwnerDisplay}"/>
+        	<f:param value="#{partBean.poolOwnerDisplay}"/>
```

Comment:
Line repeated

Reply:
i believe it is purposeful .... i can take another look but i think it's needed twice